src/frontend/src/components/AboutFooter/AboutFooter.jsx

```diff
@@ -51,6 +52,7 @@ const AboutFooter = () => {
   const classes = useStyles();
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.up('md'));
+  // const { contributor, error } = useTelescopeContributor();
```

Comment:
Modify this file only when you ship the components to display it.
Perform these tests on your local machine, but don't ship it until you have the entire work. 

The main focus of this PR is the hooks right? So keep the scope on it, ship only the hooks.